src/con_duct/__main__.py

```diff
@@ -512,14 +512,14 @@ def from_argv(
             "-l",
             "--log_level",
             default=DEFAULT_LOG_LEVEL,
-            choices=("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"),
-            help="Log level from duct operation.",
+            choices=("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG", "NONE"),
```

Comment:
```suggestion
            choices=("NONE", "CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"),
```
if to have it ordered from "nothing" to the "most".

Reply:
its currently order most to None, would you prefer to swap the whole order?